docs/modules/ROOT/partials/getting-started/ref-registry-artifact-states.adoc

```diff
@@ -0,0 +1,39 @@
+// Metadata created by nebel
+// ParentAssemblies: assemblies/getting-started/as_registry-reference.adoc
+
+[id="registry-artifact-states"]
+
+= {registry} artifact states
+These are the valid artifact states in {registry}:
+
+.{registry} artifact states
+[%header,cols=2*]
+|===
+|State
+|Description
+|`ENABLED`
+|Basic state, all the operations are available.
+|`DISABLED`
+|The artifact and its metadata is viewable and searchable using the {registry} web console, but its content cannot be fetched by any client.
+|`DEPRECATED`
+|The artifact is fully usable but a header is added to the REST API response whenever the artifact content is fetched.
+|`DELETED`
+|Artifact not available through the REST API.
```

Comment:
There is some inconsistency in this in our actual implementation... but I don't see this as a state.  I believe when an artifact is deleted we currently actually remove it - we do *not* set its "state" to `DELETED`.  I would remove this.

Reply:
You're right, the `DELETED` enum value is used in order to tell the Registry which operation to perform. I'll change this.